src/Lesson8/HomeTask2.java

```diff
@@ -0,0 +1,20 @@
+package Lesson8;
+
+public class HomeTask2 {
+    public static void main(String[] args) {
+
+        try {
```

Comment:
Try/catch should be inside divideByZero() method. And don't leave catch block empty.